capstone/src/main/webapp/form.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Form</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <script src="https://apis.google.com/js/platform.js" async defer></script>
+    <meta name="google-signin-client_id" content="">
+  </head>
+  <body>
+    <div id="content">
+      <p><h1>Quiz</h1></p>
+      <iframe src="https://docs.google.com/forms/d/e/1FAIpQLScrpUcq5Jnv6knV6K3aexWa1m972EmMzHJUvXulHCfBsJaEpw/viewform?embedded=true">Loading…</iframe>
+      <br>
+      <br>
```

Comment:
If the goal is some whitespace, CSS should handle this. Use HTML tags when it reflects the actual structure of the data, and CSS when it is purely a visual thing

Reply:
Alright, sounds good. We will delete the html tags that deal with style for now and will focus on styling the page in a future PR.